backend/docs/API.md

```
@@ -812,12 +819,18 @@ rate limiting returns `429`.
 |-------|------|-------------|
-| `type` | `ExchangeType` | `BINANCE` · `KRAKEN` |
-| `apiKey` | `string` | Exchange API key |
-| `apiSecret` | `string` | Exchange API secret |
+| `type` | `ExchangeType` | `BINANCE` · `KRAKEN` · `MERIA` |
+| `apiKey` | `string` | Exchange API key (required, max 200 chars) |
+| `apiSecret` | `string?` | Exchange API secret (max 300 chars). **Required** for `BINANCE` and `KRAKEN`; must be **omitted** for `MERIA`, which authenticates with a single read-only API key |
 
 **Response `200` — `AccountResponse`.**
 
+**Errors:**
+| Status | When |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)